lib/membrane_aac_format/remote.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Membrane.RemoteStream.AAC do
+  @moduledoc """
+  Format definition for Remote AAC Stream with known audio decoder configuration
```

Comment:
Ok, one more point here. "remote AAC stream" is not a well-known term. I can only guess what it is.
1. It should not start with capital letters
2. You should describe what kind of streams they are (depayloaded from containers with decoder config retrieved from that container?)